packages/graphback-cli/tests/workflow-test.ts

```diff
@@ -1,40 +1,31 @@
 // tslint:disable-next-line: match-default-export-name no-implicit-dependencies
 import ava, { ExecutionContext } from 'ava';
-// tslint:disable-next-line: no-var-requires no-require-imports
-const execa = require('execa');
 import { existsSync } from 'fs';
-import { DropCreateDatabaseAlways } from 'graphql-migrations';
 import { join, resolve } from 'path';
-import { connect, createDB, generate } from '../src';
+import { createDB, generate, initConfig } from '../src';
 
 const model = {
   modelName: "testSchema",
   content: `
-  type Test {
-    id: ID!
-    title: String!
-    description: String!
-  }
+    type Test {
+      id: ID
+      name: String
+    }
 `
 }
 
 ava('Test cli workflow', async (t: ExecutionContext) => {
   const basePath = resolve(`${__dirname}/../../../examples/generator-fullstack`);
   process.chdir(basePath)
+  await initConfig("testback ", { model, database: "sqlite3", client: true });
   await generate();
 
-  const db = await connect("sqlite3", { filename: "./db.sqlite" })
-
-  const databaseInitializationStrategy = new DropCreateDatabaseAlways("sqlite3", db);
-
-  await createDB(databaseInitializationStrategy);
+  await createDB();
 
   t.true(existsSync(join(basePath, "graphback.json")))
   t.true(existsSync(join(basePath, "db.sqlite")))
-  t.true(existsSync(join(basePath, "client/src/graphql/fragments/Note.ts")))
-  t.true(existsSync(join(basePath, "server/src/schema/generated.ts")))
-  t.true(existsSync(join(basePath, "server/src/resolvers/generated/note.ts")))
-
-
+  t.true(existsSync(join(basePath, "client/src/graphql/fragments/Test.ts")))
+  t.true(existsSync(join(basePath, "src/schema/generated.ts")))
```

Comment:
Any reason server was flattened?

Master still have it nested:
https://github.com/aerogear/graphback/tree/master/examples/generator-fullstack

Reply:
`initConfig` calls this:

https://github.com/aerogear/graphback/blob/a2bde3d2425dbf9a36b79cfb4802bcb46a8238c0/packages/graphback-cli/src/templates/configTemplates.ts#L83 which in turn returns these default values:
https://github.com/aerogear/graphback/blob/31e499203cb9a4d67ae60d4bc705c0ce240fa7bf/packages/graphback-cli/src/config/ConfigBuilder.ts#L11

which do not match the structure of generator-fullstack.

Reply:
Agreement. Let's revert this change. I have posted issue to fix that in CMD